gflownet/envs/README.md

```diff
@@ -11,3 +11,35 @@ By default, this redundancy is in place. However, the user may choose to manuall
 - To skip the check in specific calls to `step()`, pass it the argument `skip_mask_check=True`.
 
 Note that the mask of invalid actions indeed flags _invalid_ actions, as opposed to determining which actions are `valid`. In other words, actions flagged as invalid in the mask (`True`) must indeed be invalid, but actions marked as "not invalid" (`False`) are not necessarily valid. This may be the case if verifying the validity of action is computationally expensive, but determining a set of actions that are invalid may be affordable.
+
+## Buffer, train data and test data
+
+A train and a test set can be created at the beginning of training. The train set may be used to sample offline (backward) trajectories. The test set may be used to compute metrics during and after training. These sets may be created in different ways, specificied by the configuration variables `env.buffer.train.type` and `env.buffer.test.type`. Options for the data set `type` are
+
+- `all`: all terminating states in the output space $\mathcal{X}$ will be added - Convenient but only feasible for small, synthetic environments like the hyper-grid.
+- `grid`: a grid of points in the output space $\mathcal{X}$ - Only available in certain environments where obtaining a grid of points is meaningful. This mode also requires specifying the number of points via `env.buffer.<train/test>.n`.
+- `uniform`: points sampled uniformly output space $\mathcal{X}$ - This mode also requires specifying the number of points via `env.buffer.<train/test>.n` and the random seed via `env.buffer.<train/test>.seed`.
+
```

Comment:
There is also a `random` option, why is this not mentioned? It's also the most general one, so might actually be most relevant.

Reply:
Right, it's because I wrote this a while ago and things have changed, so thanks for pointing this out. I will add that.

Reply:
Done [here](https://github.com/alexhernandezgarcia/gflownet/pull/178/commits/87ba5e50102f35164e92fea13c63071d9c53fb3b).